deployment/cfn/worker.py

```diff
@@ -339,6 +339,9 @@ def create_auto_scaling_resources(self, worker_security_group, worker_lb):
     def get_cloud_config(self):
         return ['#cloud-config\n',
                 '\n',
+                'mounts:\n',
+                '  - [xvdf, /mnt/rwd-data, "auto", "defaults"]\n'
```

Comment:
I think that explicitly setting the following arguments is important:

```yaml
- [xvdf, /mnt/rwd-data, ext4, defaults,nofail,discard, 0, 2
```

Here `nofail` turns into `_netdev` (when mounted), which signals to the OS that this file system depends on the network and `discard` enables TRIM support for `gp2` volumes.

The `0` is to signal that the file system shouldn't be `dump(8)`'d and the `2` is for all non-root file systems (root file systems get `1`).

See also: `man fstab`